.contrib/Parser/DATAS/03 - World Drops/04 Burning Crusade/03 Weapons.lua

```diff
@@ -209,6 +209,9 @@ _.WorldDrops =
 				i(25141),	-- Halaani Hammer
 				i(31299),	-- The Oathkeeper
 				i(31322),	-- The Hammer of Destiny
+				i(25402, {	-- The Stoppable Force
+					["description"] = "Generic TBC drop.",
+				}),
```

Comment:
Don't think the description is needed here, they are all generic drops.
```suggestion
				i(25402), -- The Stoppable Force
```